### DISCUSSION TITLE
[TOOL] Keira3 public demo database (readonly)

### DISCUSSION THREAD

**Discussion by FrancescoBorzi** [azerothcore/azerothcore-wotlk#3911] [Tools]
If you want to try Keira3 using a public database where you have read-only access.

You can download the latest Keira3 version from: https://github.com/azerothcore/Keira3/releases

Then use the following DB:

```
Host: not available anymore
Port: 3306
Username: readonly
Password: readonly
Database: world
```

![image](https://user-images.githubusercontent.com/75517/77674475-f339a500-6f8b-11ea-89ac-56e486836dcd.png)

--------------------------------
Special thanks to @Viste who made this public DB available :heart:

**Comment by FrancescoBorzi** [comment]
@Viste seems no longer available

**Comment by Viste** [comment]
@FrancescoBorzi try now there were problems after the OS was updated.

**Comment by FrancescoBorzi** [comment]
@Viste now it works, thank you!

**Comment by Viste** [comment]
Can be down for today and tommorow( need to redo a few things and change the location a bit for 10G access.)
i write when all be online. Thanks for understanding and sorry :)

**Reply by Viste** [reply]
ok half of work done, today in night i run RO DB back to live

**Comment by Viste** [comment]
Get back online can someone test?

**Reply by FrancescoBorzi** [reply]
working for me, thanks @Viste !

**Comment by Viste** [comment]
Want switch OS from RHEL 8 to Rocky linux(from centos creator), so DB can be down today and tonight by Mocsow time.
Sorry if someone plan to use it today :(

**Reply by FrancescoBorzi** [reply]
go Rocky!

**Comment by Viste** [comment]
Sorry guys, I passed out to sleep and fell asleep first on the keyboard then my wife woke me up and offered to go to bed :D. In the near future, everything will come up.
But I can tell you that Rocky is awesome.

**Comment by Viste** [comment]
DB is online now

**Comment by Annamaria-CC** [comment]
@Viste 

Getting  
![image](https://user-images.githubusercontent.com/20357406/139594568-6b575db6-94dc-4f16-9ccb-dac544c73106.png)

**Comment by Viste** [comment]
@temperrr now must be ok

**Reply by FrancescoBorzi** [reply]
@Viste can you update it with AC DB 4.0.0 ?

**Reply by Viste** [reply]
> @Viste can you update it with AC DB 4.0.0 ?

yes check tommorow if its still outdated pm me in discord i check cron

**Comment by Winfidonarleyan** [comment]
```stoped because of ban on azerothcore```
-- Viste

**Comment by vurtne** [comment]
Excuse me, how can I modify the skill effect of accessories?

**Reply by FrancescoBorzi** [reply]
@vurtne this is offtopic, please reach us on Discord or open a new thread with your request
